include/odp/api/spec/pool.h

```diff
@@ -290,10 +290,16 @@ odp_pool_t odp_pool_lookup(const char *name);
 /**
  * Pool information struct
  * Used to get information about a pool.
+ * @note The difference between end_addr & start_addr
+ * will result in buffers address range belong to this pool.
```

Comment:
I'd delete this note as it's not necessary and not complete for the ODP API spec.

Reply:
OK. agreed